data/en/test_dropdown_mandatory_with_label.json

```diff
@@ -0,0 +1,47 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Dropdown Mandatory With Label",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "groups": [{
+        "id": "dropdown",
+        "title": "Dropdown Mandatory With Label",
+        "blocks": [{
+                "type": "Question",
+                "id": "dropdown-mandatory-with-label",
+                "questions": [{
+                    "type": "General",
+                    "id": "dropdown-mandatory-with-label-question",
+                    "title": "Which football team do your support?",
+                    "answers": [{
+                        "type": "Dropdown",
+                        "id": "dropdown-mandatory-with-label-answer",
+                        "mandatory": true,
+                        "label": "Please choose an option",
```

Comment:
Is this schema needed now that this label isn't going to be configurable

Reply:
The label is what is displayed above the drop-down box and not what the text for the first option displays.